frontend/apps/web/components/site-header/AccountStatusHandler.tsx

```diff
@@ -35,3 +50,65 @@ export function AccountStatusHandler(props: Props) {
     </div>
   );
 }
+
+interface TrialCountdownProps {
+  trialEndDate: number;
+  isAccountStatusValidResp: IsAccountStatusValidResponse | undefined;
+}
+
+function TrialCountdown(props: TrialCountdownProps) {
+  const { trialEndDate, isAccountStatusValidResp } = props;
+
+  const now = Date.now();
+  const daysRemaining = Math.max(
+    0,
+    Math.ceil((trialEndDate - now) / (1000 * 60 * 60 * 24))
+  );
+
+  const isExpired =
+    isAccountStatusValidResp?.accountStatus ==
+    AccountStatus.ACCOUNT_TRIAL_EXPIRED;
+  const isAlmostExpired = daysRemaining <= 3;
+
+  if (isExpired)
```

Comment:
based on this logic it doesn't look like the `isAlmostExpired` will ever actually be used.
If `isExpired` is true, then `isAlmostExpired` will always be true..

Reply:
whoops meant to clean that up, that was leftover code